### PR TITLE
Update link to Keystone Enterprise

### PR DESCRIPTION
**Pull request by benderham**
Updates the Main and Footer Navigation to include an external link to the new Keystone Enterprise page on the Thinkmill website.

Also updates the `LinkItem` to accept a `rel` and `target`.

### CODE REVIEW

File: docs/components/Footer.tsx [hunk 1]
```diff
@@ -1,19 +1,18 @@
-
 /** @jsxImportSource @emotion/react */
 
-import Link from 'next/link'
-import { type HTMLAttributes } from 'react'
+import Link from 'next/link';
+import { type HTMLAttributes } from 'react';
```
Comment (gautamsi) [inline docs/components/Footer.tsx:4]: I believe the formatting rule is not to use semi

File: docs/components/Header.tsx [hunk 2]
```diff
@@ -84,7 +84,17 @@ function useCurrentSection () {
   if (['/blog'].some(check)) return '/blog'
 }
 
-function LinkItem ({ children, href }: { children: ReactNode, href: string }) {
+function LinkItem({
```
Comment (dcousens) [inline docs/components/Header.tsx:87]:
```suggestion
function LinkItem ({
```

File: docs/components/Header.tsx [hunk 3]
```diff
@@ -84,7 +84,17 @@ function useCurrentSection () {
...
+  children,
+  href,
+  target,
+  rel,
+}: {
+  children: ReactNode,
+  href: string,
+  target?: string,
+  rel?: string
```
Comment (dcousens) [inline docs/components/Header.tsx:96]:
```suggestion
  children: ReactNode
  href: string
  target?: string
  rel?: string
```

File: docs/components/Header.tsx [hunk 4]
```diff
@@ -126,8 +138,8 @@ function FlatMenu ({
   label,
   items = [],
 }: {
-  label: string
-  items: Array<{ label: string, href: string }>
+  label: string,
+  items: Array<{ label: string; href: string }>
```
Comment (dcousens) [inline docs/components/Header.tsx:142]:
```suggestion
  label: string
  items: Array<{ label: string, href: string }>
```

**Review by dcousens — APPROVED** [review]

**Pull request merged**